docs/src/tutorial/simple_system.md

```diff
@@ -24,19 +26,54 @@ system on Spine Toolbox.
 
 ### Installation and upgrades
 
-If you haven't yet installed the tools yet, please follow the installation guides: 
-- For Spine Toolbox: [Spine Toolbox installation guide](https://github.com/spine-tools/SpineOpt.jl#installation)
-- For SpineOpt: [SpineOpt installation guide](https://github.com/spine-tools/Spine-Toolbox#installation)
+If you haven't yet installed the tools or you are not sure whether you have the latest version,
+please follow the installation/upgrade guides: 
+- For Spine Toolbox: [Spine Toolbox installation guide](https://github.com/spine-tools/Spine-Toolbox#installation)
+- For SpineOpt: [SpineOpt installation guide](https://github.com/spine-tools/SpineOpt.jl#installation)
 
-If you are not sure whether you have the latest version, please upgrade to ensure compatibility with this guide.
+## Guide
 
-- For Spine Toolbox:
-    - If installed with pipx, then use `python -m pipx upgrade spinetoolbox`
-    - If installed from sources using git, then `git pull`, `python -m pip install -U -r requirements.txt`
+### Spine Toolbox workflow
 
-- For SpineOpt: [SpineOpt upgrade guide](https://github.com/spine-tools/SpineOpt.jl#upgrading)
+The workflow for this tutorial is quite simple: A SpineOpt tool that 
+reads data from an input database, executes the simulation/optimization 
+and writes the results to an output database.
 
-## Guide
+It is almost as simple as dragging these items (i.e. Data Store and Run SpineOpt) 
+to the *Design View* and connecting them by dragging arrows between the blocks 
+but there are some things that configured:
+-   The databases need to be initialised. Once you select a database
+    you see the properties panel. Select the dialect of the database.
+    Here we choose sqlite. Then press the button 'new spine db' to 
+    create and save the database on your computer 
+    (Spine Toolbox will suggest a good folder).
+
+-   The arrows in the Toolbox view are only the available items (databases).
+    To let SpineOpt know we want to use these items, we need to go to the 
+    properties panel of Run SpineOpt and drag the available items to the 
+    tool arguments. The order of the items is first the input, then the output.
+    See below for how the property window should look like.
```

Comment:
```suggestion
    See below for how the property window should look.
```
_How_ it should _look_. _What_ it should look _like_. (At least in American English - we're weird.)